client/starwhale/base/scheduler/step.py

```diff
@@ -75,12 +75,20 @@ def get_steps_from_yaml(
         # default run index 0 handler
         job_name = job_name or "0"
         jobs = load_yaml(yaml_path)
-        if job_name in jobs:
-            job = jobs[job_name]
-        else:
-            job_index = int(job_name)
-            sorted_jobs = sorted(jobs.items())
-            job = sorted_jobs[job_index][1]
+        sorted_jobs = sorted(jobs.items())
+
+        job = None
+        try:
```

Comment:
add `Except` clause?  using `try` means there is possible exceptions

Reply:
Exception or no exception , we can use finally to print the handlers info.